R/io.R

```diff
@@ -174,13 +177,17 @@ read_civis.sql <- function(x, database = NULL, using = utils::read.csv,
 #' # Create new table, append if already exists
 #' write_civis(df, "schema.my_table", "my_database", if_exists="append")
 #'
-#' # Create new table with defined diskey / sortkeys for speed!
+#' # Create new table with defined diskey / sortkeys for speed
```

Comment:
Isn't it a generally accepted rule that "speed" is always followed by "!" ?   ;)

Reply:
we could have two `!!`